src/app/events/page.tsx

```diff
@@ -1,26 +1,53 @@
 import EventCard from "@/components/EventCard";
+import { AnimatedTitle, Title } from "@/components/Title";
 import fetchEvents from "../lib/fetchEvents";
-import SearchInput from "@/components/Search";
+import InputAndFilters from "@/components/InputAndFilters";
+import ProgramSelector from "./ProgramSelector";
 
-export default async function page({ searchParams }: { searchParams?: { query?: string } }) {
-  const query = searchParams?.query || "";
+interface PageProps {
+  searchParams?: { q?: string };
```

Comment:
query parameters aren't filtering events because the searchQuerys object has keys `program` and `location` instead of `q` if user selects a filter